src/app/issue/settings/settings.component.html

```diff
@@ -87,7 +90,7 @@ <h2>Ticket Einstellungen</h2>
     <p>Visibility Status*</p>
   </div>
   <div nz-col nzSpan="18">
-    <nz-select ngModel="extern" [(ngModel)]="visibleInput" required>
+    <nz-select ngModel="extern" [(ngModel)]="visibleInput">
       <nz-option nzValue="extern" nzLabel="Extern"></nz-option>
```

Comment:
Hier könntest du als nzValue ``true`` für extern machen (statt dem String), dann hast du es in der .ts einfacher.